cypress/e2e/orders/receiving-and-check-in/pol-with-linked-holdings-could-be-found-by-location-filtering.cy.js

```diff
@@ -0,0 +1,255 @@
+                          secondOrderLine.purchaseOrderId = secondOrderResponse.id;
+                          OrderLines.createOrderLineViaApi(secondOrderLine);
+                        });
+                        cy.loginAsAdmin({ path: TopMenu.ordersPath, waiter: Orders.waitLoading });
+                        Orders.createOrderViaApi(thirdOrder).then((thirdOrderResponse) => {
+                          thirdOrder.id = thirdOrderResponse.id;
+                          thirdOrderNumber = thirdOrderResponse.poNumber;
+                          Orders.searchByParameter('PO number', thirdOrderNumber);
+                          Orders.selectFromResultsList(thirdOrderNumber);
+                          OrderLines.addPOLine();
+                          OrderLines.selectRandomInstanceInTitleLookUP('*', 10);
+                          OrderLines.rolloverPOLineInfoforPhysicalMaterialWithFund(
+                            firstFund,
+                            '50',
+                            '1',
+                            '50',
+                            location.name,
+                          );
+                          OrderLines.backToEditingOrder();
+                          Orders.resetFilters();
+                        });
+                        Orders.createOrderViaApi(fourthOrder).then((fourthOrderResponse) => {
+                          fourthOrder.id = fourthOrderResponse.id;
+                          fourthOrderNumber = fourthOrderResponse.poNumber;
+                          Orders.searchByParameter('PO number', fourthOrderNumber);
+                          Orders.selectFromResultsList(fourthOrderNumber);
+                          OrderLines.addPOLine();
+                          OrderLines.selectRandomInstanceInTitleLookUP('*', 10);
+                          OrderLines.rolloverPOLineInfoforPhysicalMaterialWithFund(
+                            firstFund,
+                            '50',
+                            '1',
+                            '50',
+                            location.name,
+                          );
+                          OrderLines.backToEditingOrder();
+                          Orders.resetFilters();
+                        });
+                      },
+                    );
+                  });
+                },
+              );
+            });
+          });
+        });
+      });
+    });
+
+    cy.createTempUser([
+      permissions.inventoryAll.gui,
+      permissions.uiReceivingViewEditCreate.gui,
+    ]).then((userProperties) => {
+      user = userProperties;
+      cy.login(userProperties.username, userProperties.password, {
+        path: TopMenu.receivingPath,
+        waiter: Receiving.waitLoading,
+      });
+    });
+  });
+
+  after(() => {
+    cy.getAdminToken();
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C357580 Order line with linked Holdings could be found by "Location" filtering facet on "Receiving" pane (thunderjet) (TaaS)',
```

Comment:
I see that the test case on the Test rail has different id number. Is it correct?

Reply:
Fixed